langfuse/integrations.py

```diff
@@ -0,0 +1,99 @@
+import os
+import functools
+from datetime import datetime
+from dotenv import load_dotenv
+
+import openai
+from openai.api_resources import ChatCompletion, Completion
+
+from langfuse import Langfuse
+from langfuse.client import InitialGeneration
+from langfuse.api.resources.commons.types.llm_usage import LlmUsage
+
+
+load_dotenv()
+
+
+class CreateArgsExtractor:
+    def __init__(self, name=None, **kwargs):
+        self.args = {}
+        self.args["name"] = name
+        self.kwargs = kwargs
+
+    def get_langfuse_args(self):
+        return {**self.args, **self.kwargs}
+
+    def get_openai_args(self):
+        return self.kwargs
+
+
+class OpenAILangfuse:
+    def __init__(self):
+        self.langfuse = Langfuse(os.environ["LF_PK"], os.environ["LF_SK"], os.environ["HOST"])
```

Comment:
This will throw an error if key is not present afaik. Is this intended? Rather use `getenv()` and provide fallback logic in case key is not present? It's a personal preference so feel free to ignore.

Reply:
Hi, If keys are not present it is handled in `Langfuse` class 
![image](https://github.com/langfuse/langfuse-python/assets/57898986/aa5ef473-39ef-4fd4-872c-1568d2833cc4)